### PR TITLE
ACMS-1382: Fixed Password Policy config import error.

### PR DESCRIPTION
**Pull request by rajeshreeputra**
**Motivation**

Fixes #ACMS-1382

**Proposed changes**
Added patch for password_policy module.

**Alternatives considered**
NA

**Testing steps**
Follow steps from ticket.

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: modules/acquia_cms_common/composer.json [hunk 1]
```diff
@@ -98,6 +98,9 @@
             },
             "drupal/focal_point": {
                 "3162210 - Preview link accidentally closes the media library": "https://www.drupal.org/files/issues/2020-10-06/3162210-17.patch"
+            },
+            "drupal/password_policy": {
+              "\"Field field_last_password_reset is unknown\" while importing configuration on site without password_policy activated": "https://www.drupal.org/files/issues/2023-02-08/password_policy-2771129-config-import-issue-fix-140.patch"
```
Comment (chandan-singh7929) [inline modules/acquia_cms_common/composer.json:103]: Remove `\"` character.

**Review by deepakmishra2 — APPROVED** [review]
LGTM , except chandan's comment to remove "/"

File: modules/acquia_cms_common/composer.json (hunk 1 above)
Comment (apathak18) [reply modules/acquia_cms_common/composer.json:103]: @rajeshreeputra Start with a Issue Number
Syntax: "Issue#no - Issue Title": "D.O MR/Patch url"

Comment (rajeshreeputra) [reply modules/acquia_cms_common/composer.json]: done

**Comment by vishalkhode1** [comment]
Closing as it's no longer required.

**Pull request closed without merging**